vmdb/app/models/vm_redhat.rb

```diff
@@ -37,4 +37,12 @@ def self.calculate_power_state(raw_power_state)
     else             super
     end
   end
+
+  def supports_migrate?
+    false
+  end
+
+  def supports_reconfigure?
+    false
+  end
```

Comment:
@jrafanie we are good as far as templates are concerned, the check already exists that does not allow migrate/publish tasks for templates on https://github.com/ManageIQ/manageiq/blob/master/vmdb/app/controllers/application_controller.rb#L2159